carma_msgs/srv/Bind.srv

```diff
@@ -0,0 +1,12 @@
+# Bind.srv
+#
+# This is a service provided by each device driver to allow consumers
+# to inform the driver they request to bind with the given ID
+#
+
+# Request
+string id
+
+---
+
+# Response
```

Comment:
Nit: Could you double check whether if we actually need this change? If it is part of the next round of changes should we wait for this one for PR's consistency?

Reply:
This change will be needed in CAR-6069 when the mock lightbar driver is ported to ROS2, but yeah good point that this probably belongs in a separate PR. I'll go ahead and remove this

Reply:
FYI mock lightbar driver will be removed and not ported to ROS2.
https://usdot-carma.atlassian.net/browse/CAR-6069

Reply:
Got it, definitely makes sense to remove this then